packages/fxa-content-server/app/scripts/views/form.js

```diff
@@ -415,7 +415,7 @@ var FormView = BaseView.extend({
       maybeFocus();
     } else {
       // tooltipId is used to bind the invalid element
-      // with the tooltip using `aria-described-by`
+      // with the tooltip using `aria-describedby`
```

Comment:
This has been an invalid `aria` attribute that we've had in here for years that I noticed on the sign-up password balloon 😭 I went ahead and updated all references.

Reply:
Great catch! ❤️ 

 Not sure how legit this is, but this hideous mess seems to be reporting some unique aria-* values per template:
```sh
git grep -Eo "\s(aria\-.*?)\=" packages | grep -Ev "\.md: " | sort | uniq
```

| PACKAGE | ARIA-* |
|:----|:----|
packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx: | `aria-label=`
packages/fxa-admin-panel/src/components/Permissions/index.tsx: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/pair/auth_complete.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/pair/index.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/pair/success.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/partial/email-autocomplete-domains.mustache: | `aria-hidden=`
packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache: | `aria-controls=`
packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/push/completed.mustache: | `aria-label=`
packages/fxa-content-server/app/scripts/templates/push/send_login.mustache: | `aria-label=`
packages/fxa-payments-server/src/components/DialogMessage/index.tsx: | `aria-hidden=`
packages/fxa-payments-server/src/components/DialogMessage/index.tsx: | `aria-label=`
packages/fxa-payments-server/src/components/PlanDetails/index.tsx: | `aria-expanded=`
packages/fxa-payments-server/src/components/PlanDetails/index.tsx: | `aria-labelledby=`
packages/fxa-payments-server/src/components/Tooltip/index.stories.tsx: | `aria-invalid=`
packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx: | `aria-labelledby=`
packages/fxa-payments-server/src/routes/Subscriptions/index.tsx: | `aria-hidden=`
packages/fxa-payments-server/src/routes/Subscriptions/index.tsx: | `aria-label=`
packages/fxa-react/components/Footer/index.tsx: | `aria-label=`
packages/fxa-react/components/LoadingSpinner/index.tsx: | `aria-label=`
packages/fxa-settings/src/components/BentoMenu/index.tsx: | `aria-controls=`
packages/fxa-settings/src/components/BentoMenu/index.tsx: | `aria-expanded=`
packages/fxa-settings/src/components/DropDownAvatarMenu/index.tsx: | `aria-controls=`
packages/fxa-settings/src/components/DropDownAvatarMenu/index.tsx: | `aria-expanded=`
packages/fxa-settings/src/components/HeaderLockup/index.tsx: | `aria-expanded=`
packages/fxa-settings/src/components/HeaderLockup/index.tsx: | `aria-haspopup=`
packages/fxa-settings/src/components/HeaderLockup/index.tsx: | `aria-label=`
packages/fxa-settings/src/components/InputPassword/index.tsx: | `aria-hidden=`
packages/fxa-settings/src/components/InputPassword/index.tsx: | `aria-label=`
packages/fxa-settings/src/components/LinkedAccounts/LinkedAccount.tsx: | `aria-label=`
packages/fxa-settings/src/components/Nav/index.tsx: | `aria-hidden=`
packages/fxa-settings/src/components/PageAvatar/index.tsx: | `aria-label=`
packages/fxa-settings/src/components/Switch/index.tsx: | `aria-checked=`

Reply:
Thanks for checking. Those all look legit to me.

Reply:
In a whimsical twist, it's a bit of a garbage grep on my part (sorry).
It still wouldn't have detected "aria-descibed-by" (since we seem to set that via code, not via HTML) and my weak grep wouldn't catch React usages like `aria-expanded={isRevealed}`:

```sh
git grep -E "\saria-.*?=\{" | cat
```

I've been annoying enough, so here's my further hacky research into this: https://gist.github.com/pdehaan/0045a9986ca870475ff2ac247431c728

And the TLDR was:

> _I think we have the following aria-* props:_
> - `aria-checked`
> - `aria-controls`
> - `aria-describedby`
> - `aria-disabled`
> - `aria-expanded`
> - `aria-haspopup`
> - `aria-hidden`
> - `aria-invalid`
> - `aria-label`
> - `aria-labelledby`